src/components/HighlightChat/HighlightChat.tsx

```diff
@@ -78,6 +85,10 @@ const HighlightChat = () => {
   // HOOKS
   useHandleClipboardPaste();
 
+  useEffect(() => {
+    trackEvent('HL Chat Initialized', {});
+  }, []);
+
```

Comment:
A similar event is called in App.tsx, redundant?

Reply:
Removed